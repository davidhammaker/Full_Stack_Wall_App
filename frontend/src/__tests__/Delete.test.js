import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import Delete from '../components/delete';


configure({ adapter: new Adapter() });


it('displays delete button correctly', () => {
  const del = renderer.create(
    <Delete />
  );
  let tree = del.toJSON();
  expect(tree).toMatchSnapshot();
});