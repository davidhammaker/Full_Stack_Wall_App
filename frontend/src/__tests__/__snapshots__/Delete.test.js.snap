// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays delete button correctly 1`] = `
<button
  className="btn btn-sm btn-danger float-right"
  onClick={[Function]}
>
  Delete
</button>
`;
